src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneCommands/ExportAzureRmDataLakeStoreItem.cs

```diff
@@ -113,14 +104,21 @@ public int ConcurrentFileCount
             ParameterSetName = DiagnosticParameterSetName)]
         public SwitchParameter Force { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 8, Mandatory = false,
```

Comment:
@rahuldutta90 please remove the `Position` from both of the parameter attributes you are adding here

Reply:
Fixed.